tests/Rules/RelationExistenceRuleTest.php

```diff
@@ -53,8 +53,8 @@ public function testRule(): void
                 12,
             ],
             [
-                'Relation \'foo\' is not found in App\Group model.',
-                14,
+                'Relation \'foo\' is not found in App\User model.',
```

Comment:
I don't understand why these are changed to `User` It should be an error for the `Group` model.